lib/watcher/express-records.js

```diff
@@ -0,0 +1,10 @@
+var agent = require('../agent');
+
+exports.worker = function(handle) {
+  agent().on('express:usage-record', function(record) {
+    handle.send({
+      cmd: 'express:usage-record',
+      record: record
```

Comment:
I either missed it or was worried about breaking Arc.. I honestly can't remember, so probably the former.